src/modules/fw_pos_control_l1/fw_pos_control_l1_params.c

```diff
@@ -409,6 +409,20 @@ PARAM_DEFINE_FLOAT(FW_LND_AIRSPD_SC, 1.3f);
  */
 PARAM_DEFINE_FLOAT(FW_LND_THRTC_SC, 1.0f);
 
+/**
+ * Go around throttle threshold
+ *
+ * The vehicle will go around if throttle position larger than threshold
+ * during approach. it will trigger if throttle position passed from below
+ * to above threshold. 0.9 means 90% position of throttle.
```

Comment:
Suggested text:
* The vehicle will go around if the throttle position moves larger than the
* threshold during approach.  It will trigger when throttle position moves from
* below to above the threshold value.  For example, a value of 0.9 means 
* 90% throttle position.  Set to a value of 0.0 to disable.

Reply:
had modified